src/tapis-app/Files/_components/Toolbar/Toolbar.tsx

```diff
@@ -96,6 +101,12 @@ const Toolbar: React.FC<{ selectedFiles: Array<Files.FileInfo> }> = ({
           console.log('Toolbar button');
         }}
       />
+      <CreateDirModal
+        isOpen={modal === 'newfolder'}
+        toggle={() => {
```

Comment:
there should be one toggle callback for all modals that sets the state of modal to null.